src/main/java/io/confluent/idesidecar/restapi/resources/RestProxyResource.java

```diff
@@ -98,19 +94,7 @@ private void handleClusterProxy(RoutingContext routingContext, ClusterProxyConte
     process(routingContext, clusterProxyProcessor, proxyContext);
   }
 
-  private void handleRBACProxy(RoutingContext routingContext, ProxyContext proxyContext) {
-    process(routingContext, rbacProxyProcessor, proxyContext);
-  }
-
-  private void ccloudControlPlaneProxy(RoutingContext routingContext) {
-    process(routingContext, controlPlaneProxyProcessor, createCcloudProxyContext(routingContext));
-  }
-
-  private void ccloudDataPlaneProxy(RoutingContext routingContext) {
-    process(routingContext, dataPlaneProxyProcessor, createCcloudProxyContext(routingContext));
-  }
-
-  public record RBACRequest(
+  public record GenericRequest(
```

Comment:
I think you can remove this `record` after removing the OpenAPI spec-related annotations.